UI/window-basic-main.cpp

```diff
@@ -1892,7 +1892,7 @@ void OBSBasic::OBSInit()
 	} else if (isWayland) {
 		if (opt_always_on_top)
 			blog(LOG_INFO,
-			     "Always On Top not available on Wayland, ignoring…");
+			     "Always On Top not available on Wayland, ignoring...");
```

Comment:
How was this line causing a compile error?

Also, this commit is unrelated to this PR and probably should have been in a separate PR.

Reply:
> How was this line causing a compile error?
> 
> Also, this commit is unrelated to this PR and probably should have been in a separate PR.

The old character '…' causes a compile error on my visual studio 2019. So I replace it with three characters ‘.’